thinc/backends/ops.py

```diff
@@ -364,7 +364,7 @@ def get_dropout_mask(self, shape: Shape, drop: Optional[float]) -> FloatsXd:
         if drop is None or drop <= 0:
             return self.xp.ones(shape, dtype="f")
         elif drop >= 1.0:
-            return self.alloc(shape)
+           return self.alloc(shape)
```

Comment:
Nitpick: Have you run `black` on this file? There should be 4 spaces here instead of 3 :-)